File: js/stage1/S1_SW_01.js
// S1_SW_01 — Day name
function dayName(n) {
  switch (n) {
    case 1:
      return "Monday";
    case 2:
      return "Tuesday";
    case 3:
      return "Wednesday";
    case 4:
      return "Thursday";
    case 5:
      return "Friday";
    case 6:
      return "Saturday";
    case 7:
      return "Sunday";
    default:
      return null;
  }
}
// Tests
[1, 3, 7, 0, 8].forEach((d) => console.log(d, "=>", dayName(d)));

File: Python/stage1/S1_SW_01.py
# S1_SW_01 — Day name using match/case
def day_name(n):
   match n:
       case 1:
           return "Monday"
       case 2:
           return "Tuesday"
       case 3:
           return "Wednesday"
       case 4:
           return "Thursday"
       case 5:
           return "Friday"
       case 6:
           return "Saturday"
       case 7:
           return "Sunday"
       case _:
           return None
# Tests
for i in range(0, 9):
   print(i, "=>", day_name(i))